source/dynamical_systems/CMakeLists.txt

```diff
@@ -51,19 +30,6 @@ install(TARGETS ${PROJECT_NAME}
     RUNTIME DESTINATION bin)
 
 if (runtests)
-    if (APPLE)
-      add_definitions(-DGTEST_USE_OWN_TR1_TUPLE)
-      add_definitions(-D__GLIBCXX__)
-    endif (APPLE)
-
-    add_subdirectory(lib/googletest)
-
-    enable_testing()
-
-    # Include the gtest library. gtest_SOURCE_DIR is available due to
-    # 'project(gtest)' above.
-    include_directories(${gtest_SOURCE_DIR}/include ${gtest_SOURCE_DIR})
-
     add_executable(runTestLinear tests/testLinear.cpp)
     target_link_libraries(runTestLinear
       gtest
```

Comment:
We have to link `pthread` here, don't we?

Reply:
yes, this overall test structure and linking will be revised next. As far as I can tell, pthread is needed for gtest on macOS and I think for tests built using SSH container toolchain. But I would rather first combine the test sources into a few high-level test targets and link those, than go and fix it for every small test target that currently exists.